packages/components/psammead-image/README.md

```diff
@@ -13,21 +27,41 @@ const Wrapper = (src, alt, width, height) => (
 );
 ```
 
-| Prop     | Type          | Required |
-|:---------|:--------------|:---------|
-| `alt`    | string        | Yes      |
-| `height` | number/string | No       |
-| `src`    | string        | Yes      |
-| `width`  | number/string | Yes      |
+Commonly used alongside [`psammead-figure`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-figure) and [`psammead-image-placeholder`](https://github.com/BBC-News/psammead/tree/latest/packages/components/psammead-image-placeholder).
+
+```jsx
+import Figure from '@bbc/psammead-figure';
+import Image from '@bbc/psammead-image';
+import ImagePlaceholder from '@bbc/psammead-image-placeholder';
+
+const WrapperComponent = ({
+  alt,
+  ratio,
+  src,
+  width,
+}) => (
+  <Figure>
+    <ImagePlaceholder ratio={ratio}>
+      <Image alt={alt} src={src} width={width} />
+    </ImagePlaceholder>
+  </Figure>
+);
```

Comment:
It seems redundant to be pulling in Figure/ImagePlaceholder as part of the example usage here. We can just have an example of the Image in isolation.

This is better practice because otherwise if/when the API changes we have to update example code in multiple places.

(Same goes for the ImagePlaceholder readme, we can just have `<Image ...props />` in there or something, the focus is on the ImagePlaceholder.